Formula/go-jira.rb

```diff
@@ -0,0 +1,37 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  bottle do
```

Comment:
Our CI will generate all bottles and the `bottle` block, so please remove these lines.

Reply:
TIL... huh. I may look into a way to make that more clear in the contribution guide. I fought this build process for at least a day off and on :)

Reply:
> I may look into a way to make that more clear in the contribution guide

Sounds like a good idea!